tools/wheels/LICENSE_win32.txt

```diff
@@ -1,11 +1,12 @@
 
 ----
 
-This binary distribution of SciPy also bundles the following software:
+This binary distribution of SciPy can also bundle the following software
+(depending on the build):
```

Comment:
Not strictly needed, since there are no build variants on windows but keeps this file in sync with the others.

Reply:
That seems okay to me, either way works.